dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/BeamSqlPrimitive.java

```diff
@@ -67,34 +67,34 @@ public T getValue() {
   }
 
   public long getLong() {
-    return (Long) getValue();
+    return Long.valueOf(getString());
```

Comment:
take the `getLong()` for an example, the `value` is actually type of `Long`, so a cast is sufficient, why first covert to String, then parse back to Long?